src/Components/Bookings/Bookings.js

```diff
@@ -1,10 +1,182 @@
-import React from "react";
+import React, { useState, useContext, useEffect } from "react";
+import "./Bookings.css";
+import BookingsModal from "./BookingsModal/BookingsModal";
+import Backdrop from "./Backdrop/Backdrop";
+import makeRequest from ".././../Utils/index";
+import deleteBooking from "../.././Helpers/Delete";
+import editBooking from "../.././Helpers/Edit";
+import getCompletedTrips from "../../Helpers/completedTrip";
+import CompletedModal from "./CompletedTripModal/CompletedModal";
+import Authcontext from "../Context/context";
 
 const Bookings = (props) => {
+	const [modalViewing, setModalViewing] = useState(false);
+	const [bookings, setBookings] = useState([]);
+	const [selectedBooking, setSelectedBooking] = useState(null);
+	const [isEdit, setIsEdit] = useState(false);
+	const [completed, setCompleted] = useState(null);
+	const [completedTrip, setCompletedTrip] = useState([]);
+	const [selectCompletedTrip, setSelectCompletedTrip] = useState(null);
+	const [CompleteView, setCompleteView] = useState(false);
```

Comment:
what work does this variable do? I can't see where it is used in the return function

Reply:
It's for controlling the viewing of the CompletedModal. If you scroll down, you will see it.